.gitignore

```diff
@@ -39,3 +39,4 @@ CMakeLists.txt.user
 /.idea/
 # Ignore Visual Studio Code's working dir
 /.vscode/
+/.[Cc]ache*/
```

Comment:
Shouldn't be in this commit (or this pr)

But could be worth submitting a pr upstream to dolphin.

Reply:
oh right that was accidentally included because vscode was being annoying